layers/+lang/bibtex/packages.el

```diff
@@ -110,3 +112,48 @@
         "ld" 'doi-utils-add-bibtex-entry-from-doi
         "li" 'isbn-to-bibtex
         "lp" 'pubmed-insert-bibtex-from-pmid))))
+
+(defun bibtex/init-ebib ()
+  (use-package ebib
+    :defer t
+    :init
+    (spacemacs/set-leader-keys "ate" 'ebib)
+    :config
+    (setq ebib-bibtex-dialect 'biblatex)
+
+    (evilified-state-evilify-map ebib-index-mode-map
+      :mode ebib-index-mode
+      :bindings
+      "j" 'ebib-next-entry
+      "k" 'ebib-prev-entry
+      "J" 'evil-scroll-page-down
+      "K" 'evil-scroll-page-up
+
+      "/" 'ebib-search
+      "n" 'ebib-search-next)
+
+    (spacemacs/set-leader-keys-for-major-mode 'ebib-index-mode
+      "j" 'ebib-jump-to-entry
+      "k" 'ebib-kill-entry
+      "b" 'biblio-lookup)
+
+    (evilified-state-evilify-map ebib-entry-mode-map
+      :mode ebib-entry-mode)
+
+    (evilified-state-evilify-map ebib-log-mode-map
+      :mode ebib-log-mode)
+
+    (use-package ebib-biblio
+      :after (ebib biblio)
+      :config
+      (evilified-state-evilify-map biblio-selection-mode-map
+        :mode biblio-selection-mode
+        :bindings
+        "e" 'ebib-biblio-selection-import
+        (kbd "C-j") 'biblio--selection-next
```

Comment:
these two `biblio` biindings should be added in `biblio` package's init function.

Reply:
I am not sure about the `ebib-biblio-selection-import` case, because that more is an ebib feature.
But for the `biblio--selection-next` I agree with you...